Exercises.Delegate/Exercises.Delegate/Point.cs

```diff
@@ -0,0 +1,107 @@
+﻿using System;
+using System.Collections;
+
+namespace Exercises.Delegate
+{
+    public class Point : IComparable
+    {
+
+        public enum PointRepresentation { Polar, Rectangular }
+
+        private double r, a;
+
```

Comment:
Please look through styling rules: order of class members, number of new lines between them, class example which I shared with you via skype. Correct all the places in all files.